Constitution.txt

```diff
@@ -1,490 +1,264 @@
+5: It is the Right of all People to practice their religion freely and openly; and the Responsibility of all People to avoid interfering in the religious practice of others, and, in the practice of their religion, to respect the Rights of other People.
 
-5: Sections 1 and 2 shall take effect on the 15th day of October following the ratification of this article.
+6: It is the Right of all People to be considered innocent of any crime, unless such crime shall be proven in a court of law, as decided by unanimous decision of a jury of at least twelve disinterested peers, and for such a trial to be carried out with reasonable speed in public, and for the result of such a trial to be respected in perpetuity; and to have access to legal Representation provided by the court or the prosecution, when the accused Person is unable to retain such Representation by their own means; and to be reasonably compensated for time served on a jury; and the Responsibility of all People to serve on such juries when called upon, and in such service adhere to the Truth, the Rights of the People, and the Law; and in such cases as the Law is in conflict with Truth or the Rights of the People, to judge in favor of Truth and the Rights of the People, against that unjust Law.
 
-6: This article shall be inoperative unless it shall have been ratified as an amendment to the Constitution by the legislatures of three-fourths of the several States within seven years from the date of its submission.   ratified #20
+Article VI (Article 6 - Limitations on the Government with respect to the Rights of the People)
 
-Article [XXI] (Amendment 21 - Repeal of Eighteenth Amendment)
-1: The eighteenth article of amendment to the Constitution of the United States is hereby repealed. affects 16
+1: No person acting on behalf of the Government of these United States, the Government of any State, or of any subservient government authority to the Government of these United States or of any State, shall act in a way which intentionally violates any of the Rights of the People. Just compensation shall be awarded from the treasuries of the offending parties to those persons whose Rights are found to have been violated by any of the above parties; and such compensation shall be doubly awarded to the victims if any of the offending parties is demonstrated in a court of law to have acted with intent to violate the Rights, with half of that compensation to be paid directly by the individuals whose intent was so demonstrated.
 
-2: The transportation or importation into any State, Territory, or possession of the United States for delivery or use therein of intoxicating liquors, in violation of the laws thereof, is hereby prohibited.
+2: When adjudicating violations of the Rights of the People, the Rights shall be taken in order of precedence as listed in Article V.
```

Comment:
This description of hierarchy is outside of the Declaration of Rights and Responsibilities, because it refers to government action. This also allows for the hierarchy to be updated if the People believe the order is wrong. Having such a hierarchy is necessary, because if, for example, a religious practice requires the violation of another's property rights, it should be recognized as explicitly incompatible with the American Faith.